.buildkite/verify_node.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- :node: Current node version is $(node --version)"
+
+# shellcheck source=/dev/null
+source "$NVM_DIR/nvm.sh" --no-use
```

Comment:
Adding more debug logs 🙂 
```suggestion
echo "NVM_DIR is: $NVM_DIR"
source "$NVM_DIR/nvm.sh" --no-use
```